src/libs/actions/ReportActions.js

```diff
@@ -67,7 +72,31 @@ function isReportMissingActions(reportID, maxKnownSequenceNumber) {
         || reportActionsMaxSequenceNumbers[reportID] < maxKnownSequenceNumber;
 }
 
+/**
+ * Get the count of deleted messages upto a sequence number of a report
+ * @param {Number|String} reportID
+ * @param {Number} lastSequenceNumber
```

Comment:
Similarly, this isn't really necessarily the "last" sequence number, just any sequence number given, so I think a slightly clearer variable name would be just `sequenceNumber`.

Reply:
Correct. Updating ...